### PR TITLE
Initial scratchpad entity

### PR DESCRIPTION
**Pull request by gbuisson**

### CODE REVIEW

File: src/ctim/schemas/bundle.cljc [hunk 1]
```diff
@@ -66,20 +69,20 @@
             :description "a list of `Tool`")
    (f/entry :verdicts (f/set-of Verdict)
             :description "a list of `Verdict`")
-   (f/entry :data-tables (f/set-of DataTable)
-            :description "a list of `DataTable`")))
+   #_(f/entry :data-tables (f/set-of DataTable)
+              :description "a list of `DataTable`")))
```
Comment (msprunck) [inline src/ctim/schemas/bundle.cljc]: Why did you disable the `:data-tables` entity?

File: src/ctim/schemas/data_table.cljc [hunk 2]
```diff
@@ -40,7 +40,7 @@
 (def Datum
   "A generic data object, this is really limited to the types
   defined in ColumnType"
-  f/any)
+  f/any-str)
```
Comment (msprunck) [inline src/ctim/schemas/data_table.cljc]: Update `column-type` if it is no longer possible to put an Observable

File: src/ctim/schemas/scratchpad.cljc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      :cljs [flanders.core :as f :refer-macros [def-entity-type def-eq]])))
+
+(def type-identifier "scratchpad")
+
+(def-eq ScratchpadTypeIdentifier type-identifier)
+
+(def scratchpad-desc
+  "Describes a CTIM Scratchpad which works like a structured gist")
+
+(def-map-type Update
```
Comment (msprunck) [inline src/ctim/schemas/scratchpad.cljc]: I don't see any use of this type

**Comment by hshahman** [comment]
Verified in rel-v0.3

**Pull request merged**